README.md

```diff
@@ -15,6 +15,18 @@ file][config].
 
 [config]: http://eslint.org/docs/user-guide/configuring#using-configuration-files
 
+### Channels
+
+There are a few major versions of ESLint out there. The latest major version
+is 4. This engine provides a few channels for major versions of ESLint.
+
+This branch is for the `latest` (i.e. default) channel provides ESLint 3
+support.
+
+For ESLint 4 please see [`channels/eslint-4`][] branch.
```

Comment:
small grammar nit: please change this to: `For ESLint 4, please see the [channels/eslint-4][] branch.`